src/main/java/net/tropicraft/core/common/config/TropicraftConfig.java

```diff
@@ -0,0 +1,29 @@
+package net.tropicraft.core.common.config;
+
+import com.google.common.collect.Lists;
+import net.minecraftforge.common.ForgeConfigSpec;
+
+import java.util.List;
+import java.util.UUID;
+
+public class TropicraftConfig {
+    public static ForgeConfigSpec.ConfigValue<List<? extends String>> portalEnchanterWhitelist;
+
+    public static void setupConfig(ForgeConfigSpec.Builder builder){
+        builder.push(" User-specific settings");
+//        builder.pop();
+
+        portalEnchanterWhitelist = builder.comment(" List of UUIDs (not usernames) of users that can use the Portal Enchanter. This tool has the ability to create portal's any where the player right clicks. NOTE: Make sure you use the version of UUID that has dashes")
```

Comment:
I believe that a more dumb down version, one that just makes a portal, should be craft-able / obtainable to allow for players to better position the portal within the future or like flint and steel, generate the portal water in some way.

If such is going to be only obtainable thru creative, it is still a good idea to have a way to prevent bad actors from using this tool on others a a method of griefing.